src/style.css

```diff
@@ -0,0 +1,96 @@
+:root {
+  --grey: #3d4e61;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.body {
+  margin: 0;
+}
+
+.form {
+  margin: 20px 0;
+}
+
+.box {
+  width: 100%;
+  font-family: inherit;
+  color: var(--grey);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+}
+
+.box__big {
+  height: 70px;
+  line-height: 70px;
+  font-size: 16px;
+  background: url(images/Search.svg) no-repeat 26px 25px/19px;
+  padding-left: 62px;
+}
+
+.box__small {
+  font-size: 14px;
+  height: 42px;
+  background: url(images/Search.svg) no-repeat 13px 15px/11px;
+  padding-left: 33px;
+}
+
+.box__big::placeholder {
+  color: var(--grey);
+}
+
+.box__small::placeholder {
+  color: var(--grey);
```

Comment:
If you have the same rule for both modifiers put it inside the whole `.box::placeholder` selector.

Reply:
add font-weight to placeholder, it shoud not become bolder on focus, only input text